### DISCUSSION TITLE
Be able to enable/disable gc2wiimote in game

### DISCUSSION THREAD

**Discussion by creedof69** [Aurelio92/RVLoader#33] [Ideas]
Hello, I just want to submit that It would be great if We could enable/disable GC2Wiimote while the game is already started. I explain myself. For example in the game Resident Evil 4 Wii, We can play with the Wiimote+Nunchuk, the CC and the GC controller, but We HAVE TO make the game check We use the Wiimote+Nunchuk. I tried to use RvLoader to skip this check screen but setting RvLoader to CC doesn't skip the screen and playing the Wiimote gameplay with a GC controller is not worth (B button is "back" in menu and "aiming" in game). I don't know if It would be technically possible to do this request, by pressing a button combination, the Wii eject button or something, but it would be appreciate in certain cases like this one when We doesn't have Wiimote working anymore.
Thanks you for reading.

**Comment by GameCube4Life** [comment]
Brother, just play the Game Cube version with Widescreen on. its a better experience.

**Reply by creedof69** [reply]
... No ?
The GameCube version can not display the widescreen there is always 2 top and bottom black bars, the GameCube version doesn't have the Ada stories separate ways and the others, and in the GameCube version the red light of guns is really invisible, I don't know why so, when you want to shoot the little blue medals far away this is impossible.
So no, I don't want to play the GameCube version, I already tried it.

**Reply by GameCube4Life** [reply]
ok
